packages/cmake.rb

```diff
@@ -52,13 +52,13 @@ def self.build
   end
 
   # Failed tests:
-  # BundleUtilities
-  # GeneratorExpression
-  # RunCMake.CMakeRelease
-  # RunCMake.file-GET_RUNTIME_DEPENDENCIES
-  # def self.check
-  #   system 'ninja -C builddir test || true'
-  # end
+  # BundleUtilities (armv7l,x86_64)
+  # CTest.UpdateGIT (x86_64)
+  # CustomCommand (armv7l,x86_64)
+  # RunCMake.CMakeRelease (armv7l,i686,x86_64)
+  def self.check
+    system 'ninja -C builddir test || true'
+  end
```

Comment:
Tests pass now?  Awesome!

Reply:
I listed the tests which pass and which don't...